### DISCUSSION TITLE
sidebar/sanfene/nixi

### DISCUSSION THREAD

**Discussion by giscus** [itwanger/tobebetterjavaer-giscus#98] [Announcements]
# sidebar/sanfene/nixi

面试前必读系列！包括 Java 基础（JavaSE）、Java 集合框架、Java 并发编程（Java 多线程）、Java 虚拟机（JVM）、Spring、Redis、MyBatis、MySQL、操作系统、计算机网络、RocketMQ 等等。

https://tobebetterjavaer.com/sidebar/sanfene/nixi.html

**Comment by moonZ000** [comment]
看看

**Comment by 547377507** [comment]
感谢分享~

**Comment by debugzhao** [comment]
大佬牛批

**Comment by chenli0109** [comment]
pdf咋获取

**Comment by 3546514206** [comment]
啦啦啦啦啦

**Comment by l674964417** [comment]
感谢大佬分享

**Comment by wanghang521** [comment]
大佬，pdf怎么获取？

**Comment by emmawas-afk** [comment]
第一天来咯

**Comment by rushPeng** [comment]
大佬，为啥你文章的图片，都加载不了呢?
